.github/workflows/windows-e2e.yml

```diff
@@ -146,6 +146,31 @@ jobs:
           name: crc-e2e-gh
           path: "crc-e2e.tar"
 
+      - name: Check SSH connection to host
```

Comment:
As in the other PR, some context to explain why we need this would be nice to have. I'm guessing this is some sanity check to debug https://github.com/crc-org/crc/issues/4034

Reply:
Thanks @cfergeau , I ran these changes on my fork by setting the branch to default: [workflow run here](https://github.com/jsliacan/crc/actions/runs/7913916516). And we figured that maybe it's good to put the repeats into the `crc-support` image that we're using to install CRC on the host. The actions in the img are those that the issue is related to (restart after installing CRC). So solution should be coupled with it too. I will close (or amend) this PR when we test it and confirm it's done. Marking WIP now. Thank you!